### DISCUSSION TITLE
Updates WingetUI

### DISCUSSION THREAD

**Discussion by Breizil29** [Devolutions/UniGetUI#924] [Q&A]
hello everyone,

![WingetUI](https://user-images.githubusercontent.com/81489376/232584009-4e7297dc-3f42-4db9-aad3-358727a4a5bf.png)

![ccleaner](https://user-images.githubusercontent.com/81489376/232584110-5d1a073e-2c7b-4662-b9fd-095b66735cfd.png)

**Comment by marticliment** [comment]
This is an issue with winget-cli, not with WingetUI. While they get this fixed, you can blacklist this package

**Comment by Breizil29** [comment]
Ok, thank you.
